koji-prometheus-exporter.py

```diff
@@ -317,7 +337,17 @@ def scrape():
         'Reported capacity of all koji hosts by channel',
         labels=HOST_LABELS,
     )
+
+    koji_hosts_last_update_family = GaugeMetricFamily(
+        'koji_hosts_last_update',
+        'Counter of last update from host',
```

Comment:
Hm, not a counter.

Reply:
It's a gauge?

Reply:
It should be a counter.  Its what I get for putting down code for a month and picking it up.